.travis.yml

```diff
@@ -1,44 +1,176 @@
-env:
-   - CABALVER=1.18 GHCVER=7.4.1 # Debian 7
-   - CABALVER=1.20 GHCVER=7.6.3 # Debian 8
-   - CABALVER=1.18 GHCVER=7.8.4
-   - CABALVER=1.22 GHCVER=7.10.3
-   - CABALVER=1.24 GHCVER=8.0.2
-   - CABALVER=1.24 GHCVER=8.2.2
-   - CABALVER=2.2 GHCVER=8.4.3
-   - CABALVER=2.4 GHCVER=8.6.3
-   - CABALVER=head GHCVER=head
-
+# This Travis job script has been generated by a script via
+#
+#   haskell-ci '--ghc-head' 'Glob.cabal'
+#
+# For more information, see https://github.com/haskell-CI/haskell-ci
+#
+# version: 0.3.20190429
+#
+language: c
+dist: xenial
+git:
+  # whether to recursively clone submodules
+  submodules: false
+cache:
+  directories:
+    - $HOME/.cabal/packages
+    - $HOME/.cabal/store
+before_cache:
+  - rm -fv $CABALHOME/packages/hackage.haskell.org/build-reports.log
+  # remove files that are regenerated by 'cabal update'
+  - rm -fv $CABALHOME/packages/hackage.haskell.org/00-index.*
+  - rm -fv $CABALHOME/packages/hackage.haskell.org/*.json
+  - rm -fv $CABALHOME/packages/hackage.haskell.org/01-index.cache
+  - rm -fv $CABALHOME/packages/hackage.haskell.org/01-index.tar
+  - rm -fv $CABALHOME/packages/hackage.haskell.org/01-index.tar.idx
+  - rm -rfv $CABALHOME/packages/head.hackage
 matrix:
-   allow_failures:
-      - env: GHCVER=head CABALVER=head
```

Comment:
Heh, I didn't realize I had this here. I guess it didn't work since the failures were not, in fact, allowed by Travis CI. I've come to rely on that behaviour somewhat since I like getting monthly e-mails from Travis CI telling me whether there's something upcoming in GHC HEAD that breaks things. I wonder if there's a way of still getting those notifications even with the builds having such failures not fail the entire build?

Reply:
The last time I checked there weren't easy way. GitHub API does support non-binary responses, but Travis doesn't let one set them simply.